damilola.code-workspace

```diff
@@ -0,0 +1,14 @@
+{
+	"folders": [
```

Comment:
Remove this file from a folder.

Reply:
Learn how to gitignore some things 
https://git-scm.com/docs/gitignore